src/app/core/services/units.service.ts

```diff
@@ -86,6 +86,12 @@ export class UnitsService implements OnDestroy {
       { measure: 'g/min', description: "Gallons per minute"},
       { measure: 'g/h', description: "Gallons per hour"}
     ] },
+    { group: 'Economy', units: [
+      { measure: 'm/m3', description: "Meters per cubic meter (base)"},
+      { measure: 'nm/l', descriotion: "Nautical Miles per liter"},
```

Comment:
I think this should be **description** for the next view lines

Reply:
DOH